src/Compilers/VisualBasic/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.vb

```diff
@@ -1035,8 +1082,9 @@ BC31535: Friend assembly reference 'WantsIVTAccess' is invalid. Strong-name sign
 
 #Region "Signing"
 
-    <ConditionalFact(GetType(WindowsDesktopOnly))>
-    Public Sub MaxSizeKey()
+    <ConditionalTheory(GetType(DesktopOnly), Reason:="TODO: find out why this won't load on CoreClr")>
```

Comment:
>TODO [](start = 54, length = 4)

This appears to be a CoreClr bug. Tracking down a good repro. Will add a tracking link here.

Reply:
Filed for followup https://github.com/dotnet/coreclr/issues/21723